tests/test-generate-docs.sh

```diff
@@ -5,22 +5,45 @@ die() { set +v; echo "$*" 1>&2 ; sleep 1; exit 1; }
 
 # Test field-descriptions.yaml and field-types.yaml:
 
-for ATTR in 'description' 'type' 'entity' 'assay'; do
+ATTR_LIST='description type entity assay schema'
+for ATTR in $ATTR_LIST; do
   PLURAL="${ATTR}s"
   [ "$PLURAL" == 'entitys' ] && PLURAL='entities'
   REAL_DEST="docs/field-${PLURAL}.yaml"
   TEST_DEST="docs-test/field-${PLURAL}.yaml"
+  echo "Checking $REAL_DEST"
 
   REAL_CMD="src/generate_field_yaml.py --attr $ATTR > $REAL_DEST"
   TEST_CMD="src/generate_field_yaml.py --attr $ATTR > $TEST_DEST"
 
   mkdir docs-test || echo "Already exists"
   eval $TEST_CMD || die "Command failed: $TEST_CMD"
   diff -r $REAL_DEST $TEST_DEST \
-    || die "Update needed: $REAL_CMD; $LOOP"
+    || (
+      for ATTR in $ATTR_LIST; do
+        REAL_DEST="docs/field-${PLURAL}.yaml"
+        REAL_CMD="src/generate_field_yaml.py --attr $ATTR > $REAL_DEST"
+        echo $REAL_CMD
+      done
+      die "Update needed: Easiest to update all files at once."
+    )
```

Comment:
A little unclear reading the diff. What's the purpose of the nested loop over `ATTR_LIST`?

Reply:
If one of these files needs updating (perhaps there's a new field) they usually all need updating... So we dump all the commands to the console so the user can easily copy-paste and run the whole batch.

Let me add a comment.